comdb2/cdb2.py

```diff
@@ -128,48 +128,79 @@
     Make sure to carefully read :ref:`String and Blob Types` on that page.
 """
 
+from __future__ import annotations
+
```

Comment:
```suggestion
```

Same here, I don't think this is necessary.

Reply:
It's not _necessary_, again, but without it I wouldn't be able to use annotations like
```py
def column_names(self) -> list[str]:
```
and would need to instead use:
```py
def column_names(self) -> "list[str]":
```
or
```py
def column_names(self) -> List[str]:
```

And likewise I'd need to change:
```py
    def execute(
        self, sql: str | bytes, parameters: Mapping[str, Value] | None = None
    ) -> Handle:
```
to be
```py
    def execute(
        self, sql: str | bytes, parameters: Optional[Mapping[str, Value]] = None
    ) -> "Handle":
```

Do you strongly prefer avoiding `from __future__ import annotations`?

Reply:
I thought that both of the typing features you mention here were already available in Python 3.8